src/main/java/io/cdap/plugin/sfmc/source/MarketingCloudInputFormat.java

```diff
@@ -71,7 +71,8 @@
    * @param conf the plugin conf
    * @return Collection of MarketingCloudObjectInfo containing table and schema.
    */
-  private static List<MarketingCloudObjectInfo> fetchTableInfo(SourceQueryMode mode, MarketingCloudSourceConfig conf) {
+  public static List<MarketingCloudObjectInfo> fetchTableInfo(SourceQueryMode mode,
+                                                                 MarketingCloudSourceConfig conf) {
```

Comment:
Fix indent. You can use this Intellij settings file - https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard. Please check all files modified in the PR

Reply:
Fixed the indents.

Reply:
hm looks like this is still not fixed

Reply:
please fix indent